Formula/readline.rb

```diff
@@ -21,6 +22,10 @@ class Readline < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}"
+    # There is no termcap.pc in the base system, so we have to comment out
+    # the corresponding Requires.private line otherwise pkg-config will
+    # consider the readline module unusable
+    system "sed", "-i", "-e", "/^Requires.private: / s/^/#/", "readline.pc"
```

Comment:
Let's use `inreplace` instead of `system "sed"`.
I believe it should be:

```
    inreplace "readline.pc", /^Requires.private: .*/, ""
```

Reply:
> Let's use `inreplace` instead of `system "sed"`.
> I believe it should be:
> 
> ```
>     inreplace "readline.pc", /^Requires.private: .*/, ""
> ```

I didn't know about `inreplace` :)

I've tweaked the call slightly:

```
inreplace "readline.pc", /^(Requires.private: .*)$/, "# \\1"
```

so that the original line is commented out instead of dropped outright.

Based on other uses of `inreplace` I've seen in the repository, eg. in `Formula/mariadb.rb`, it should work.